### ISSUE TITLE
xai decoder issue gradcam gradient access

### ISSUE THREAD

**Issue by muellerdo** [frankkramer-lab/aucmedi#233]
Inputs and outputs of gradient access model part of the gradient-based XAI methods have to be reworked due to keras 3 now enforces keras tensors and do not allow lists anymore.

Fix:
```py
        # Gradient model construction
        layer_output = self.model.get_layer(self.layerName).output
        model_output = self.model.output
        if isinstance(model_output, list):
            outputs = [layer_output] + model_output
        else:
            outputs = [layer_output, model_output]

        gradModel = tf.keras.models.Model(inputs=self.model.inputs,
                         outputs=outputs)
```

todo: make PR

**Comment by muellerdo** [comment]
Fixed
